seed/models/analysis_property_views.py

```diff
@@ -31,3 +31,6 @@ class AnalysisPropertyView(models.Model):
     # that are applicable to this specific property.
     # For results not specific to the property, use the Analysis's parsed_results
     parsed_results = JSONField(default=dict, blank=True)
+
+    def get_output_files(self):
+        return self.analysisoutputfile_set
```

Comment:
As far as I understand it I need to give the serializer a method to populate the custom field, rather than a data source:
"output_files = AnalysisOutputFileSerializer(source='get_output_files', many=True)"

I can define this function in the serializer but I figured it should live on the model it's related to.

Reply:
Oh, I meant that  the serializer might be able to take `analysisoutputfile_set` directly.
```
output_files = AnalysisOutputFileSerializer(source='analysisoutputfile_set', many=True)
```